pixoo/__init__.py

```diff
@@ -99,7 +100,7 @@ def __init__(self, address, size=64, debug=False, refresh_connection_automatical
         if self.simulated:
             self.__simulator = Simulator(self, simulation_config)
 
-    def clear(self, rgb=Palette.BLACK):
+    def clear(self, rgb: object = Palette.BLACK) -> object:
```

Comment:
Why add typing (only) here?

Reply:
`def clear(self, rgb: object = Palette.BLACK) -> object:`
I don't know why in my main branch it was already like this